core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -1278,6 +1278,44 @@ class KafkaConfigTest {
     KafkaConfig.fromProps(props)
   }
 
+  @Test
+  def testImplicitAllBindingControllerListenersCanBeAdvertisedForKRaftCombined(): Unit = {
+    val props = new Properties()
+    props.setProperty(KRaftConfigs.PROCESS_ROLES_CONFIG, "controller")
```

Comment:
The name of the method doesn't look correct since the process role is just controller.

Reply:
Updated case name. Thanks.